test/test.js

```diff
@@ -18,557 +19,653 @@ var describeStatusMessage = !/statusMessage/.test(http.IncomingMessage.toString(
   ? describe.skip
   : describe
 
-describe('finalhandler(req, res)', function () {
-  describe('headers', function () {
-    it('should ignore err.headers without status code', function (done) {
-      request(createServer(createError('oops!', {
-        headers: { 'X-Custom-Header': 'foo' }
-      })))
-        .get('/')
-        .expect(shouldNotHaveHeader('X-Custom-Header'))
-        .expect(500, done)
-    })
-
-    it('should ignore err.headers with invalid res.status', function (done) {
-      request(createServer(createError('oops!', {
-        headers: { 'X-Custom-Header': 'foo' },
-        status: 601
-      })))
-        .get('/')
-        .expect(shouldNotHaveHeader('X-Custom-Header'))
-        .expect(500, done)
-    })
+const servers = [
+  ['http', createHTTPServer],
+  ['http2', createHTTP2Server]
+]
```

Comment:
Let's make full testing

Reply:
No one test was deleted, just remove `describe('finalhandler(req, res)', () => {})` wrapper to avoid extra nesting (reading purpses)